src/apis/vaults.ts

```diff
@@ -16,7 +17,7 @@ export interface VaultsAPI {
     mapReplaceRequests(vaultId: AccountId): Promise<Map<H256, ReplaceRequest>>;
     getPagedIterator(perPage: number): AsyncGenerator<Vault[]>;
     get(vaultId: AccountId): Promise<Vault>;
-    getVaultCollateralization(vaultId: AccountId, newCollateral?: DOT): Promise<number | undefined>;
+    getVaultCollateralization(vaultId: AccountId, newCollateral?: BalanceWrapper): Promise<number | undefined>;
```

Comment:
Let's keep the previous method signature and package `BalanceWrapper` ourselves.